src/server.c

```diff
@@ -2986,6 +2986,11 @@ struct redisCommand *lookupCommandOrOriginal(robj **argv ,int argc) {
     return cmd;
 }
 
+/* Commands arriving from the master client or AOF client, should never be rejected. */
+int mustObeyClient(client *c) {
```

Comment:
i'd rather not add a flag here.. i envision that some day we'll add other conditions to this function.
regarding the term, both words you suggested are too long (they break my OCR).
since this is a "command" we're talking about, we need to "obey" :smile: